src/pages/index.js

```diff
@@ -45,15 +50,16 @@ const IndexPage = () => (
 
     <section className="featured-project">
       <Project
-        title="GoVoteGSO App"
+        title="Go Vote App"
```

Comment:
Tim had left a comment to change the reference to go vote to GoVoteGSO, which is the title on the site. This should be changed back.

Reply:
Ah, I copypasta the updated project component from projects page that has the inactive link fix and when I did that it reverted the title. Title fixed in both locations with new commit.